Resources/views/MediaAdmin/edit.html.twig

```diff
@@ -50,8 +67,7 @@ file that was distributed with this source code.
                             <table class="table">
                                 <tr>
                                     <th>{{ 'label.size'|trans({}, 'SonataMediaBundle') }}</th>
-                                    <td>{{ object.width|number_format_decimal }}px x {{ object.height|number_format_decimal }}px
-                                        {% if object.size > 0 %}({{ object.size|number_format_decimal }}o){% endif %}</td>
+                                    <td>{{ object.width }} x {{ object.height }} {% if object.size > 0 %}({{ macros.bytesToSize(object.size) }}){% endif %}</td>
```

Comment:
See [the changelog](https://github.com/symfony/symfony/blob/5129c4cf7e294b1a5ea30d6fec6e89b75396dcd2/src/Symfony/Bundle/TwigBundle/CHANGELOG.md#270), I'm right since 2.7.0